src/.nuget/packages.config

```diff
@@ -1,6 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.DotNet.BuildTools" version="1.0.24-prerelease" />
+  <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-2" />
+  <package id="Microsoft.DotNet.TestHost" version="1.0.3-prerelease" />
```

Comment:
These should only be in the test-runtime\packages.config they shouldn't need to be ehere. Only thing that should be here is the BuildTools (and the OpenCover/ReportGenerator for now I will be moving them at some point as well).